### DISCUSSION TITLE
I can see the console in browser, but "Default" project never loads

### DISCUSSION THREAD

**Discussion by locohost** [supabase/supabase#6868] [Questions]
This is my first time here. I humbly apologize if I'm doing this wrong...

Its certain I'm doing something wrong. I installed Supabase (Docker) on a VM on my Proxmox server. In my browser (Brave) on my Win10 laptop, I can go to http://supabase:3000/. Supabase is in my hosts file and resolves to the VM IP. In the browser, I do see the Supabase main screen as expected. 

However, when I click on the included "Default" project it loads forever. Literally. It never finishes loading. 

What did I mess up?

**Comment by subhasundardass** [comment]
I have same issue

**Comment by zlever01** [comment]
Any updates? I'm having the same issue.

**Reply by madpelsm** [reply]
I found that this is most likely due to supebase/postgres, apparently it won't work on ARM yet and rebuilding it yourself might help. I have not tried this. Instead I turned to appwrite, which does work out of the box and has kind of the same features as supabase.

**Reply by zlever01** [reply]
I was trying to host on a VPS. Checked the specs and it was an AMD processor. It just hung on 'Connecting to Default Project'. I spun up a VPS with an intel processor got the same result.

**Comment by Aman-Aalam** [comment]
I just installed the self-hosted version on an Ubuntu machine running on AMD processor - first of all everything worked in the first go, so HUGE KUDOS to the team.

I can load the studio but default project takes forever to connect.

Any idea what's going on?

The server is another machine accessible on my network.
